backend/migrations/Version20250917182707.php

```
@@ -0,0 +1,59 @@
+                feed_item_contents TO feed_item_content,
+                tags               TO feed_tag,
+                user_feeds         TO feed_user,
+                user_feed_items    TO feed_user_item,
+                weather_forecasts  TO weather_forecast,
+                notes              TO user_note,
+                checklist_items    TO user_checklist_item,
+                user_settings      TO user_setting
+        ');
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)